dependencies.gradle

```diff
@@ -3,46 +3,48 @@ ext {
             // Build tools and SDK
             buildTools      : "29.0.3",
             compileSdk      : 29,
+            gradle          : "3.6.1",
             jetpack         : "1.1.0",
-            kotlin          : "1.3.50",
+            kotlin          : "1.3.61",
```

Comment:
why not 71?

Reply:
The was a compilation warning because Gradle plugin v3.6.1 was compiled with Kotlin v1.3.61.